frontend/appflowy_flutter/lib/plugins/document/presentation/editor_page.dart

```diff
@@ -304,4 +311,15 @@ class _AppFlowyEditorPageState extends State<AppFlowyEditorPage> {
     }
     return const (false, null);
   }
+
+  Future<void> _initializeShortcuts() async {
+    defaultCommandShortcutEvents;
```

Comment:
?

Reply:
This value is lazily evaluated, so it needs to be called before initializing the shortcuts. Any suggestions to improve it? It seems unintuitive.

Reply:
Removed.

Reply:
Mayur, can you add a `TODO(Xazin): Refactor lazy initialization`, I will look into it.

Reply:
Sure